modin/pandas/test/test_series.py

```diff
@@ -1733,6 +1733,26 @@ def test_dt(timezone):
     df_equals(modin_series.dt.end_time, pandas_series.dt.end_time)
     df_equals(modin_series.dt.to_timestamp(), pandas_series.dt.to_timestamp())
 
+    def dt_with_empty_partition(lib):
+        # For context, see https://github.com/modin-project/modin/issues/5112
+        df_a = lib.DataFrame({"A": [lib.to_datetime("26/10/2020")]})
+        df_b = lib.DataFrame({"B": [lib.to_datetime("27/10/2020")]})
+        df = lib.concat([df_a, df_b], axis=1)
+        # BaseOnPython should have a single partition after the concat
+        # because it uses pandas for the concat.
+        if isinstance(df, pd.DataFrame) and get_current_execution() == "BaseOnPython":
+            assert df._query_compiler._modin_frame._partitions.shape == (1, 1)
```

Comment:
```suggestion
```

I think that while it's important to check we have 1x2 partitioning after `eval`, it's not important to check the partitioning before the `eval`.